docs/rules/new-cap.md

```diff
@@ -76,6 +88,14 @@ Array of uppercase-starting function names that are permitted to be used without
 
 If provided, it must be an `Array`. The default values will continue to be excluded when `capIsNewExceptions` is provided.
 
+### `newIsCapWildcardExceptions`
+
+Array of lowercase library names where all child methods belong to are permitted to be used with `new` operator
```

Comment:
i wonder if it would be a better API to allow this to be an array of RegExp source strings - ie, pass it into `new RegExp()` and use the resulting regular expression to do the test.

That way things like `^` and `$` and `$` etc all work, and the plugin doesn't have to implement any of them :-)